README.md

```diff
@@ -146,6 +146,18 @@ See [#chart](https://iextrading.com/developer/docs/#chart) for detailed document
 
 If a symbol cannot be found an [IEX::Errors::SymbolNotFound](lib/iex/errors/symbol_not_found_error.rb) exception is raised.
 
+### InvalidOption
+
+If an invalid option is passed to `Chart.get`, an [IEX::Errors::InvalidOption](lib/iex/errors/invalid_option_error.rb) exception is raised.
```

Comment:
I think it's enough to say "An IEX:...InvalidOption is raised when invalid options are specified."